### DISCUSSION TITLE
Can't download any of release from Github on home wifi

### DISCUSSION THREAD

**Discussion by lazydevil013** [community/community#11731] [Other Feature Feedback, Questions, & Ideas]
I have been having an issue where I can't download releases of any software from github while on my home internet. If I were to switch to my mobile Hotspot, it would download normally. It isn't a matter of parental control restrictions either. Whenever I try to download a release, whatever web browser I'm using just throws some generic can't connect to server error.

I can download from source fine. I have tried this on multiple devices, but I get the same result.

My ISP is Plusnet Communication Pvt. Ltd
![Screenshot (37)](https://user-images.githubusercontent.com/99993364/154750410-e707339d-2d8c-492c-afbe-2b59ecc572f3.png)

**Comment by mattplate** [comment]
Im on the same here. I try to install CasaOS from my wifi and nothing, but with my hotspot, no problem.
Any idea?

**Comment by prajwaldhage** [comment]
I have the same problem with CasaOS. does anyone got solution for this?

**Comment by dimexy247** [comment]
Same here
